_data/apps.yml

```diff
@@ -1109,3 +1109,12 @@
     - "game"
     - "strategy"
   icon: "istrolid.png"
+-
+  name: "JANDI"
+  description: "Team Communication for Businesses"
+  website: "https://www.jandi.com"
+  keywords:
+    - "messanger"
+    - "collaboration"
+  icon: "jandi.png"
+  featuredOnHomepage: true
```

Comment:
@hohobabyp we aren't currently adding new features apps at this time to the homepage, I've removed this line in https://github.com/electron/electron.atom.io/commit/6ce7717e21ea275f2d04f18a63eeb29f7266dc1e

It is still displayed on http://electron.atom.io/apps/